EXPLAINER.md

```diff
@@ -169,9 +184,10 @@ document.onvisibilitychange = event => {
 ## Detailed design discussion
 
 ### Low level API vs standardized NDEF records only
-NDEF comes with a set of standard records with data encoded in particular ways,
-like text which also encode the encoding and language name, and even
-smart-poster, which can contain icons and descriptions in multiple languages.
+NDEF (NFC Data Exchange Format) comes with a set of standard records with
+data encoded in particular ways, like text which also encode the encoding
+and language name, and even smart-poster, which can contain icons and
+descriptions in multiple languages.
```

Comment:
Mention media types? Though not much used. URL records are among the most used, though.